src/frontend/github-oauth.ts

```diff
@@ -148,14 +151,39 @@ export default (opts: OAuthOptions): GithubOAuth => {
 		checkGithubAuth: async (req: Request, res: Response, next: NextFunction) => {
 			const traceLogsEnabled = await booleanFlag(BooleanFlags.TRACE_LOGGING, false);
 			const tracer = new Tracer(logger.child(opts), "checkGithubAuth", traceLogsEnabled);
+			try {
+				const { githubToken } = req.session;
```

Comment:
This is the only place that's acceptable to access the `githubToken` from the session.  Everything else needs to call `res.locals.githubToken`